openasip/opset/base/base.cc

```diff
@@ -1273,9 +1273,15 @@ OPERATION(EQF)
 
 TRIGGER
     if (BWIDTH(1) == FLT_WORD_SIZE && BWIDTH(2) == FLT_WORD_SIZE) {
-        IO(3) = (!isnan(FLT(1)) && !isnan(FLT(2)) && FLT(1) == FLT(2)) ? 1 : 0;
+        IO(3) =
+            (!std::isnan(FLT(1)) && !std::isnan(FLT(2)) && FLT(1) == FLT(2))
+                ? 1
```

Comment:
Yep, this doesn't look nice :) Can you describe the original style in clang-format options?